pkg/artifactory/resource_artifactory_local_alpine_repository.go

```diff
@@ -4,37 +4,39 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
-var alpineLocalSchema = mergeSchema(baseLocalRepoSchema, map[string]*schema.Schema{
-	"primary_keypair_ref": {
-		Type:     schema.TypeString,
-		Optional: true,
-		Description: "Used to sign index files in Alpine Linux repositories. " +
-			"See: https://www.jfrog.com/confluence/display/JFROG/Alpine+Linux+Repositories#AlpineLinuxRepositories-SigningAlpineLinuxIndex",
-	},
-}, compressionFormats)
-
 func resourceArtifactoryLocalAlpineRepository() *schema.Resource {
+	const pkt = "alpine"
+
+	var alpineLocalSchema = mergeSchema(getBaseLocalRepoSchema(pkt), map[string]*schema.Schema{
+		"primary_keypair_ref": {
+			Type:     schema.TypeString,
+			Optional: true,
+			Description: "Used to sign index files in Alpine Linux repositories. " +
+				"See: https://www.jfrog.com/confluence/display/JFROG/Alpine+Linux+Repositories#AlpineLinuxRepositories-SigningAlpineLinuxIndex",
+		},
+	}, compressionFormats)
+
+	type AlpineLocalRepo struct {
+		LocalRepositoryBaseParams
+		PrimaryKeyPairRef string `hcl:"primary_keypair_ref" json:"primaryKeyPairRef"`
```

Comment:
The default packer will actually do the `hcl` properly for you. You only need be explicit for overrides

Reply:
It is part of the original code. Type declaration has been brought inside the function `resourceArtifactoryLocalAlpineRepository` as part of code clean up. Therefore showing as a change/commit.